### PR TITLE
Update comments referring to AAD to now refer to Microsoft Entra

### PR DESCRIPTION
**Pull request by TrevorJoelHarris**
## Description

AAD has been renamed to Microsoft Entra. See [here](https://www.microsoft.com/en-us/security/business/identity-access/azure-active-directory) for more details. I've updated that name in our comments and docs.

### Main changes in the PR:

1. Updated comments/docs to refer to Microsoft Entra instead of AAD.

### Validation performed:

None

### Unit Tests added:

No

### End-to-end tests added:

No

### Change file added:

Yes

### CODE REVIEW

File: packages/teams-js/src/public/people.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ export namespace people {
     title?: string;
 
     /**
-     * Optional; AAD ids of the users to be pre-populated in the search box of people picker control
+     * Optional; Microsoft Entra ids of the users to be pre-populated in the search box of people picker control
```
Comment (erikadoyle) [inline packages/teams-js/src/public/people.ts:99]: Should "ids" be capped `IDs` for consistency? (Like in line 29)

File: packages/teams-js/src/public/people.ts [hunk 2]
```diff
@@ -120,7 +120,7 @@ export namespace people {
    */
   export interface PeoplePickerResult {
     /**
-     * user object Id (also known as aad id) of the selected user
+     * user object Id (also known as Microsoft Entra id) of the selected user
```
Comment (erikadoyle) [inline packages/teams-js/src/public/people.ts:123]: Same here (ID)

File: packages/teams-js/src/public/profile.ts [hunk 3]
```diff
@@ -67,13 +67,13 @@ export namespace profile {
    * The set of identifiers that are supported for resolving the persona.
    *
    * At least one is required, and if multiple are provided then only the highest
-   * priority one will be used (AadObjectId > Upn > Smtp).
+   * priority one will be used (AadObjectId > Upn > Smtp). AAD is now known as "Microsoft Entra"
```
Comment (erikadoyle) [inline packages/teams-js/src/public/profile.ts:70]: AAD is now known as "Microsoft Entra ID"

File: packages/teams-js/src/public/interfaces.ts [hunk 4]
```diff
@@ -530,7 +530,7 @@ export interface Context {
    * @deprecated
    * As of 2.0.0, please use {@link app.UserInfo.id | app.Context.user.id} instead
    *
-   * The Azure AD object id of the current user.
+   * The Microsoft Entra object id of the current user.
```
Comment (erikadoyle) [inline packages/teams-js/src/public/interfaces.ts:533]: ID

File: packages/teams-js/src/public/app.ts [hunk 5]
```diff
@@ -385,7 +385,7 @@ export namespace app {
     licenseType?: string;
 
     /**
-     * A value suitable for use when providing a login_hint to Azure Active Directory for authentication purposes.
+     * A value suitable for use when providing a login_hint to Microsoft Entra for authentication purposes.
```
Comment (erikadoyle) [inline packages/teams-js/src/public/app.ts:388]: Microsoft Entra ID

File: packages/teams-js/src/public/authentication.ts [hunk 6]
```diff
@@ -46,12 +46,12 @@ export namespace authentication {
   /**
    * Initiates an authentication flow which requires a new window.
    * There are two primary uses for this function:
-   * 1. When your app needs to authenticate using a 3rd-party identity provider (not Azure Active Directory)
-   * 2. When your app needs to show authentication UI that is blocked from being shown in an iframe (e.g., Azure Active Directory consent prompts)
+   * 1. When your app needs to authenticate using a 3rd-party identity provider (not Microsoft Entra)
```
Comment (erikadoyle) [inline packages/teams-js/src/public/authentication.ts:49]: Microsoft Entra ID

File: packages/teams-js/src/public/interfaces.ts [hunk 7]
```diff
@@ -508,7 +508,7 @@ export interface Context {
    * @deprecated
    * As of 2.0.0, please use {@link app.UserInfo.loginHint | app.Context.user.loginHint} instead
    *
-   * A value suitable for use as a login_hint when authenticating with Azure AD.
+   * A value suitable for use as a login_hint when authenticating with Microsoft Entra.
```
Comment (erikadoyle) [inline packages/teams-js/src/public/interfaces.ts:511]: Microsoft Entra ID

File: packages/teams-js/src/public/interfaces.ts (hunk 4 above)
Comment (TrevorJoelHarris) [reply packages/teams-js/src/public/interfaces.ts:533]: I think this means to change `object id` to `object ID`

**Review by lakhveerk — APPROVED** [review]
LGTM

**Review by Ella-ly — APPROVED** [review]

**Review by erikadoyle — APPROVED** [review]

**Pull request merged**